stratisd.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Stratis daemon
 Documentation=man:stratisd(8)
+After=dbus.socket
```

Comment:
This unfortunately leads to a dependency cycle:

```
Aug 20 12:13:38 dev systemd[1]: local-fs.target: Found ordering cycle on local-fs-pre.target/start
Aug 20 12:13:38 dev systemd[1]: local-fs.target: Found dependency on stratisd.service/start
Aug 20 12:13:38 dev systemd[1]: local-fs.target: Found dependency on dbus.socket/start
Aug 20 12:13:38 dev systemd[1]: local-fs.target: Found dependency on sysinit.target/start
Aug 20 12:13:38 dev systemd[1]: local-fs.target: Found dependency on ldconfig.service/start
Aug 20 12:13:38 dev systemd[1]: local-fs.target: Found dependency on local-fs.target/start
Aug 20 12:13:38 dev systemd[1]: local-fs.target: Job local-fs-pre.target/start deleted to break ordering cycle starting with local-fs.target/start
```

Reply:
I was warned by @mulkieran that this might not be the way to go. I'm reevaluating this piece.